docs/notebooks/longrangecnot/LongRangeCNOTgates.ipynb

```diff
@@ -0,0 +1,2139 @@
+    "$$\n",
+    "\n",
+    "where $\\rho_{\\mathcal{U}^\\dagger\\circ\\xi}$ is the density operator obtained from the channel $\\mathcal{U}^\\dagger\\circ\\xi$ via the Choi-Jamoiłkawski isomorphism\n",
+    "\n",
+    "$$\n",
+    "\\rho_{\\mathcal{U}^\\dagger\\circ\\xi} = \\bigl(I\\otimes(\\mathcal{U}^\\dagger\\circ\\xi)\\bigr)(|\\psi_+\\rangle\\langle\\psi_+|)\n",
+    "$$\n",
+    "\n",
+    "and where $|\\psi_+\\rangle$ is the maximally entangle state\n",
+    "\n",
+    "$$\n",
+    "|\\psi_+\\rangle = \\frac{1}{\\sqrt{d}}\\sum_{i=0}^{d-1}|i\\rangle \\otimes |i\\rangle.\n",
+    "$$\n",
+    "In our specific situation, where $\\mathcal{U}$ is a unitary channel, the entanglement fedility of $\\mathcal{U}^\\dagger\\circ\\xi$ can be written in terms of the *process fidelity* of the two Choi states $\\rho_\\mathcal{U}$ and $\\rho_{\\xi}$ as follows:\n",
+    "$$\n",
+    "\\mathcal{F}_{ent}(\\mathcal{U}^\\dagger\\circ\\xi) = \\mathcal{F}_{proc}(\\rho_\\mathcal{U}, \\rho_{\\xi}) := \\mathcal{F}(\\rho_\\mathcal{U}, \\rho_{\\xi})\n",
+    "$$\n",
+    "and so we see via Proposition 1 of Horodecki et al. [ref]  that \n",
+    "$$\n",
+    "\\mathcal{F}_{avg}(\\mathcal{U},\\xi) = \\mathcal{F}_{avg}(\\mathcal{U}^\\dagger\\circ\\xi) = \\frac{d\\mathcal{F}_{ent}(\\mathcal{U}^\\dagger\\circ\\xi) + 1}{d+1} = \\frac{d\\mathcal{F}(\\rho_\\mathcal{U}, \\rho_{\\xi}) +1}{d+1}\n",
+    "$$\n",
+    "Calculating the process fidelity between two states can now be achieved via Monte Carlo state certification. \n",
+    "\n",
+    "As per [ref] a direct implementation of the quantum Monte Carlo state certification would\n",
+    "prepare a maximally entangled state $|\\psi_+\\rangle$, apply $\\xi$ to half of\n",
+    "the system, and then measure random Pauli operators on all\n",
+    "qubits. A more practical approach consists of preparing the\n",
+    "complex conjugate of random product of eigenstates of local\n",
+    "Pauli operators (corresponding to the resulting state after half\n",
+    "of the entangled state is measured destructively), applying the\n",
+    "transformation $\\xi$ to the system, and finally measuring a random Pauli operator on each qubit. This can be seen from the following equality:\n",
+    "\n",
+    "$$\n",
+    "\\mathrm{Tr}\\bigl[(P_i\\otimes P_j\\otimes P_k\\otimes P_l)(I\\otimes\\xi)(|\\psi_+\\rangle\\langle\\psi_+|)\\bigr]\n",
+    "= \\frac{1}{d}\\mathrm{Tr}\\bigl[(P_k\\otimes P_l)\\cdot \\xi(P_i^*\\otimes P_j^*)\\bigl]\n",
+    "$$\n",
+    "\n",
+    "The following three experiments use the modified and simplified version of Monte Carlo state certification combined with the relations derived above to calculate the average gate fedility of the channel $\\xi$. For more details see [ref] and associated references."
```

Comment:
This part goes a bit too deep into the fidelity calculation protocol, I would just add a reference to it in the notebook. Overall, it would be nice to have the code that implements the method in our repo (or in qiskit) as it is fairly general along with an how-to-guide that goes into the theory described here and shows how to use it

Reply:
This was written because the original paper's explanation is confusing and mixes up terminology which is confusing to the reading who does not know this stuff. This should eventually go into a separate notebook about State Certification and measuring average gate fidelity that this notebook can reference. As there is not time now I have instead moved this section to an Appendix of the notebook.